viewer/test-utilities/src/createPointCloudModel.ts

```diff
@@ -41,8 +49,5 @@ export function createPointCloudModel(modelId: number, revisionId: number): Cogn
   const nodeWrapper = new PotreeNodeWrapper(pointCloudOctree, [], Symbol('dummy'), { classificationSets: [] });
 
   const pointCloudNode = new PointCloudNode(potreeGroup, nodeWrapper);
-
-  const pointCloudModel = new CognitePointCloudModel(modelId, revisionId, pointCloudNode);
-
-  return pointCloudModel;
+  return pointCloudNode;
```

Comment:
You can just return the pointcloud node directly. No need to bind it to variable here.